src/WellIO.jl

```diff
@@ -1,5 +1,20 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+           read(f, String)
+        end
```

Comment:
Notice how indentation is incorrect here. Please make sure your editor is using spaces for tabs, and use a consistent number of spaces everywhere.